### ISSUE TITLE
WCAG 4.1.2 tests 22 and 23: false positives?

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#227]
[Test document 3](https://github.com/veraPDF/veraPDF-wcag-algs/files/9119848/test-document-3.pdf) fails WCAG 4.1.2 tests 22 (_Invalid List element_) and 23 (_Invalid List item element_). Both errors suggest that the algorithm does not recognise the tagged lists as lists. Visually they are clearly intended as lists, although they do run over two pages.

Is this a false positive? Or is this a case of a confusing error message where a more accurate message would be "two list items are tagged as a single one", or something like that?

Note that this issue is probably related to[ issue #133](https://github.com/veraPDF/veraPDF-wcag-algs/issues/133).

**Comment by MaximPlusov** [comment]
In this case more accurate message would be "two list items are tagged as a single one".
